qt_ui/windows/mission/flight/settings/QGeneralFlightSettingsTab.py

```diff
@@ -21,6 +21,7 @@
 
 class QGeneralFlightSettingsTab(QFrame):
     flight_size_changed = Signal()
+    pilots_changed = Signal()
```

Comment:
This one could be removed if we would define a signal in `QFlightSlotEditor`, and then use that signal to pass on to `QFlightStartType`, but even that could be circumvented by doing something like:
```
self.flight_slot_editor = QFlightSlotEditor(package_model, flight, game.game)
qfst = QFlightStartType(package_model, flight)
self.flight_slot_editor.pilots_changed.connect(qfst.on_pilot_selected)
```

Reply:
Done 👍

Reply:
Made a small adjustment after your changes, we can also get rid of the Signal parameter in `QFlightStartType` and connect the signal in `QGeneralFlightSettingsTab`